### PR TITLE
Remove ChromeOS support

### PR DESCRIPTION
**Pull request by McGiverGim**
After discussing this for several months, we think we need to remove the support for Chrome OS. It is consuming a lot of maintenance time because it needs a different API than the other platforms and we limit with some versions, etc.

This does not changes some things that we are doing in a "chrome" way to a Node/NW way, because I suppose we will need to maintain the old version for Android/Cordova support. This job can be done in more incremental way in future PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,7 @@
 
 Betaflight Configurator is a crossplatform configuration tool for the Betaflight flight control system.
 
-It runs as an app within Google Chrome and allows you to configure the Betaflight software running on any [supported Betaflight target](https://github.com/betaflight/betaflight/tree/master/src/main/target).
-
-There is also now a standalone version available, since Google Chrome Apps are getting deprecated on platforms that aren't Chrome OS. [Downloads are available in Releases.](https://github.com/betaflight/betaflight-configurator/releases)
+It runs as an application under different operating systems and allows you to configure the Betaflight software running on any [supported Betaflight target](https://github.com/betaflight/betaflight/tree/master/src/main/target) and [downloads are available in Releases.](https://github.com/betaflight/betaflight-configurator/releases)
```
Comment (mikeller) [inline README.md:9]: This target list is very outdated - it should be removed.

File: README.md [hunk 2]
```diff
@@ -84,12 +72,12 @@ To build or release only for one specific platform you can append the plaform af
...
-* **Linux** use `yarn gulp <task-name> --linux64`
-* **Windows** use `yarn gulp <task-name> --win32`
-* **ChromeOS** use `yarn gulp <task-name> --chromeos`
+* **Linux** use `yarn gulp <task-name> --linux64` 
+* **Windows** use `yarn gulp <task-name> --win32` 
 * **Android** use `yarn gulp <task-name> --android`
 
-You can also use multiple platforms e.g. `yarn gulp <taskname> --osx64 --linux64`.
+
+You can also use multiple platforms e.g. `yarn gulp <taskname> --osx64 --linux64`. Others platforms like `--win64`, `--linux32` and `--armv7` can be used too, but they are not officially supported, so use them at your own risk.
```
Comment (mikeller) [inline README.md:80]: 'Other'

**Comment by McGiverGim** [comment]
Fixed as per review!

**Comment by mikeller** [comment]
I know not your code, but the SonarCloud fixes are very simple, so you could as well do them. ;-)

**Comment by McGiverGim** [comment]
I was thinking the same, but I was not too sure... Fixed!!!

File: gulpfile.js [hunk 3]
```diff
@@ -112,10 +112,10 @@ gulp.task('default', debugBuild);
 
 // Get platform from commandline args
 // #
-// # gulp <task> [<platform>]+        Run only for platform(s) (with <platform> one of --linux64, --linux32, --armv7, --osx64, --win32, --win64, --chromeos or --android)
+// # gulp <task> [<platform>]+        Run only for platform(s) (with <platform> one of --linux64, --linux32, --armv7, --osx64, --win32, --win64, or --android)
 // #
 function getInputPlatforms() {
-    const supportedPlatforms = ['linux64', 'linux32', 'armv7', 'osx64', 'win32','win64', 'chromeos', 'android'];
+    const supportedPlatforms = ['linux64', 'linux32', 'armv7', 'osx64', 'win32','win64', 'android'];
```
Comment (haslinghuis) [inline gulpfile.js:118]:
```suggestion
    const supportedPlatforms = ['linux64', 'linux32', 'armv7', 'osx64', 'win32', 'win64', 'android'];
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: gulpfile.js (hunk 3 above)
Comment (haslinghuis) [reply gulpfile.js:118]: Hopefully implementing linter will suppress my pointless reviews complaining about styling issues :trollface:

Comment (McGiverGim) [reply gulpfile.js:118]: I don't think they are pointless, I like to have the code right. I will change it tomorrow. Thanks for the review.

**Comment by mikeller** [comment]
Sorry, this needs a rebase now.

File: gulpfile.js (hunk 3 above)
Comment (chmelevskij) [reply gulpfile.js]: Sorry @haslinghuis, I should have expressed my thoughts in better way 😉

**Comment by McGiverGim** [comment]
Rebased!!!

File: gulpfile.js (hunk 3 above)
Comment (haslinghuis) [reply gulpfile.js]: no worries 😃

**Review by mikeller — APPROVED** [review]

**Comment by mikeller** [comment]
@McGiverGim: Not sure what is wrong here - tests seem to be failing on MacOS after it, despite a number of retries. I'll try again tomorrow to see if this is a transient problem.

**Comment by McGiverGim** [comment]
I think I found the culprit... I removed by error in the `.travis.yml` the:
```yaml
addons:
  chrome: stable
```
without suspecting it was being used by our tests 🤦‍♂️

**Comment by mikeller** [comment]
Lol yes, the tests are trying to use Chrome. Thanks for fixing.

**Review by mikeller — APPROVED** [review]

**Pull request merged**